koji_wrapper/base.py

```diff
@@ -7,14 +7,16 @@
 """
 
 import koji
-import operator
+
 
 class KojiWrapperBase(object):
 
-    def __init__(self, url='', topurl='', session=None):
+    def __init__(self, url='', topurl='',
```

Comment:
Would changing this order break things for existing callers?

Reply:
No, this is why I used names parameters, so you can  pass them in any order you like, and defaults if at all possible, so existing calls (though I doubt there are any yet, as we only have v 0.1.0 so far) should mostly be unaffected on updates